iceoryx_dds/include/iceoryx_dds/gateway/dds_to_iox.hpp

```diff
@@ -27,22 +28,22 @@ namespace dds
 ///
 /// @brief DDS Gateway implementation for the DDS to iceoryx direction.
 ///
-template <typename channel_t = iox::dds::Channel<iox::popo::Publisher, iox::dds::data_reader_t>,
-          typename gateway_t = iox::dds::DDSGatewayGeneric<channel_t>>
+template <typename channel_t = iox::popo::Channel<iox::popo::Publisher, iox::dds::data_reader_t>,
+          typename gateway_t = iox::popo::GatewayGeneric<channel_t>>
```

Comment:
`GatewayGeneric` shall be in a separate namespace

Reply:
`iox::gateway` or `iox::gw` ?

Reply:
iox::gateway would be more specific.